src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationFunctionsTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Functions.FunctionArgValidators;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Interpreter;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+using Xunit;
+using static Microsoft.PowerFx.Core.Localization.TexlStrings;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class MutationFunctionsTests : PowerFxTest
+    {
+        private readonly Engine _engine;
+        private readonly ParserOptions _opts = new ParserOptions { AllowsSideEffects = true };
+
+        public MutationFunctionsTests()
+        {
+            _engine = new Engine(new PowerFxConfig());
+
+            var symbol = new SymbolTable();
+
+            symbol.EnableMutationFunctions();
+
+            _engine.Config.SymbolTable = symbol;
+        }
+
+        [Theory]
+        [InlineData("Collect(t, Blank()).F2", "Collect(t, Blank()).Display2", true)]
+        [InlineData("Collect(t, {Display1:1/0}).F0", "Collect(t, {Display1:1/0}).Display0", true)]
+        [InlineData("Collect(t, {F0:0}).F3", "Collect(t, {Display0:0}).F3", true)]
+        [InlineData("Collect(t, {Display2:false}).F3", "Collect(t, {Display2:false}).F3", true)]
+        [InlineData("Collect(t, {F1:1}).F0;Collect(t, {F2:false}).F0", "Collect(t, {Display1:1}).Display0;Collect(t, {Display2:false}).Display0", true)]
+        [InlineData("Patch(t, First(t), {F1:2})", "Patch(t, First(t), {Display1:2})", true)]
```

Comment:
> First(t) [](http://example.com/codeflow?start=29&length=9)

All the Patch use First(t). 
Can you add a test for arg1 that uses display names? (ie, a Lookup expression